src/common/types.ts

```diff
@@ -60,6 +60,17 @@ export interface UserAlias {
   alias_label: string
 }
 
+/**
+ * Prioritization options for user merging when multiple users are found.
+ * This is an ordered array - if more than one user matches from a prioritization, merging will not occur.
+ * {@link https://www.braze.com/docs/api/endpoints/user_data/post_user_identify#identifying-users-by-email}
+ */
+export type PrioritizationOption =
```

Comment:
Can we rename this to `Prioritization`?

```suggestion
export type Prioritization =
```